packages/common/src/store/pages/profile/reducer.ts

```diff
@@ -43,86 +43,169 @@ const initialState = {
 
   profileMeterDismissed: false,
 
-  [FollowType.FOLLOWERS]: { status: Status.LOADING, userIds: [] },
-  [FollowType.FOLLOWEES]: { status: Status.LOADING, userIds: [] },
-  [FollowType.FOLLOWEE_FOLLOWS]: { status: Status.LOADING, userIds: [] }
+  [FollowType.FOLLOWERS]: { status: Status.IDLE, userIds: [] },
+  [FollowType.FOLLOWEES]: { status: Status.IDLE, userIds: [] },
+  [FollowType.FOLLOWEE_FOLLOWS]: { status: Status.IDLE, userIds: [] }
 }
 
+const initialState = { currentUser: null, entries: {} }
+
 const actionsMap = {
   [FETCH_PROFILE](state, action) {
-    if (action.fetchOnly) return state
+    const { fetchOnly, shouldSetLoading, handle, userId } = action
+    if (fetchOnly) return state
+
+    const profileState = state.entries[handle] ?? initialProfileState
 
     const newState = {
```

Comment:
Could rename this to `newEntry` for clarity

Reply:
totes